t/08-autotest.t

```diff
@@ -316,9 +316,13 @@ subtest 'load test successfully when CASEDIR is a relative path' => sub {
 };
 
 subtest python => sub {
-    stderr_like {
+    my $python_module_regex = qr{scheduling pythontest tests/pythontest};
+    my $python_version_regex = qr{Using python version};
+
+    combined_like {
         lives_ok { autotest::loadtest('tests/pythontest.py') } 'can load test module'
-    } qr{scheduling pythontest tests/pythontest.py}, 'python pythontest module referenced';
+    } qr{$python_module_regex+$python_version_regex}, 'python pythontest module referenced';
+
```

Comment:
why not running `autotest::loadtest('tests/pythontest.py')` for each assertion?

one for testing that `scheduling pythontest ...` exists. one for the python version.

Reply:
@josegomezr that was my initial idea, for some reason this isn't captured properly

Reply:
I'm no regex expert, I understand the regex match you're using is:

`#(?^:(?^:scheduling pythontest tests/pythontest)+(?^:Using python version))#`

against:

```
[2023-10-17T09:26:39.922954Z] [debug] [pid:2061] scheduling pythontest tests/pythontest.py
```

The first part of the regex  `$python_module_regex` matches, but the second it's just immediately after, it's expecting `$python_version_regex` (or `Using python version`) to be immediately after the module name.

I think you're printing two lines instead. See this regex101:

https://regex101.com/r/jsB2gf/1